diff --git a/ptdump/CMakeLists.txt b/ptdump/CMakeLists.txt
index c885a5b..9d267fb 100644
--- a/ptdump/CMakeLists.txt
+++ b/ptdump/CMakeLists.txt
@@ -48,4 +48,4 @@ add_executable(ptdump
   ${PTDUMP_FILES}
 )
 
-target_link_libraries(ptdump libipt)
+target_link_libraries(ptdump ipt)
diff --git a/ptdump/src/ptdump.c b/ptdump/src/ptdump.c
index 72c5bed..4a2b18a 100644
--- a/ptdump/src/ptdump.c
+++ b/ptdump/src/ptdump.c
@@ -207,10 +207,12 @@ static int version(const char *name)
 {
 	struct pt_version v = pt_library_version();
 
+#if 0
 	printf("%s-%d.%d.%d%s / libipt-%" PRIu8 ".%" PRIu8 ".%" PRIu32 "%s\n",
 	       name, PT_VERSION_MAJOR, PT_VERSION_MINOR, PT_VERSION_BUILD,
 	       PT_VERSION_EXT, v.major, v.minor, v.build, v.ext);
 	return 0;
+#endif
 }
 
 static int parse_range(const char *arg, uint64_t *begin, uint64_t *end)
